packages/playground/src/stores/stats.ts

```diff
@@ -0,0 +1,13 @@
+import { defineStore } from "pinia";
+import { computed } from "vue";
+
+import { useAsync } from "@/hooks";
+const url = window.env.STATS_URL || "https://stats.grid.tf";
+
+export const useRequestStore = defineStore("request-store", () => {
+  const res = useAsync(() => fetch(url + "/api/stats-summary").then(resp => resp.json()), {
+    init: true,
+  });
+  const data = computed(() => res.value.data);
+  return { data };
+});
```

Comment:
`useRequestStore` is a very generic name shouldn't we call it something like `useStatsSummary`

also we are computing `stats` array in 2 places why don't we return it here directly?